storage/src/mmr/benches/build_mmr.rs

```diff
@@ -0,0 +1,26 @@
+use commonware_cryptography::{Digest, Sha256};
+use commonware_storage::mmr::mem::Mmr;
+use criterion::{criterion_group, Criterion};
+
+fn bench_build_mmr(c: &mut Criterion) {
+    let element = Digest::from_static(&[100u8; 32]);
+    c.bench_function("build_mmr", |b| {
```

Comment:
You need to populate this field with `module_path!()/params (if any)` like: https://github.com/commonwarexyz/monorepo/blob/c876abb2c6c379bf9dc04304c4ffa54186a1e06e/cryptography/src/bls12381/benches/aggregate_public_keys.rs#L8

Reply:
done